src/components/contacts/index.tsx

```diff
@@ -0,0 +1,20 @@
+import Table from '../common/table';
+import { columns, sortingOptions } from './helper';
+import { contacts, total } from '../../store/contacts/selector';
+import { deleteContactAsync, fetchContactAsync } from '../../store/contacts/thunk';
+
+const ContactsTable = () => {
+  return (
+    <Table
+      columns={columns}
+      fetch={fetchContactAsync}
```

Comment:
`fetch` -> `fetchAction`

Reply:
fix: renamed to fetchAction in Contacts, Tickets and Table folders